user/src/com/google/gwt/user/server/rpc/RPC.java

```diff
@@ -629,23 +693,54 @@ private static int getRpcVersion() throws SerializationException {
    * @param wasThrown if true, the object being returned was an exception thrown
    *          by the service method; if false, it was the result of the service
    *          method's invocation
-   * @return a string that encodes the response from a service method
+   * @return a string that encodes the response from a service method, or {@code null} if a protocol version of
```

Comment:
I agree that public methods are even more relevant for updating the Javadoc comments. I found a number of public methods, such as

```
  public static String invokeAndEncodeResponse(Object target, Method serviceMethod, Object[] args,
      SerializationPolicy serializationPolicy, int flags) throws SerializationException {
```
but they didn't have a Javadoc at all, so I didn't feel it would be appropriate to introduce one just for this change. Are you recommending to do so?

Where can I find and edit the release notes?